pyntcloud/io/las.py

```diff
@@ -11,26 +15,64 @@ def convert_location_to_dtype(data, dtype_str):
 
 
 def get_color_dtype(data, column_names):
-    color_data_types = [data["points"][column_name].dtype for column_name in column_names]
-    assert len(set(color_data_types)) == 1
-    return color_data_types[0]
+    has_color = True
+    for column_name in column_names:
+        if column_name not in data["points"]:
+            has_color = False
+            break
+    if has_color:
+        color_data_types = [data["points"][column_name].dtype for column_name in column_names]
+        assert len(set(color_data_types)) == 1
```

Comment:
Try to replace assertions with a concrete exception raising in order to inform the user what exactly went wrong.

Ref on reasons behind:
https://stackoverflow.com/a/20076295/6076911

Reply:
Thank you for pointing that out!